tests/mutations/conftest.py

```diff
@@ -47,4 +48,16 @@ class Mutation:
 
 @pytest.fixture
 def mutation(db):
-    return utils.generate_query(mutation=Mutation)
+    mutation = Mutation
+
+    if settings.GEOS_IMPORTED:
+        from ..types import GeoField, GeoFieldInput, GeoFieldPartialInput
+
+        @strawberry.type
+        class GeoMutation(Mutation):
+            createGeoField: GeoField = mutations.create(GeoFieldInput)
+            updateGeoFields: List[GeoField] = mutations.update(GeoFieldPartialInput)
```

Comment:
Those should be written in snake_case, the schema converter is responsible for converting it to camelCase

Reply:
Done f5ac99b01ce7eb40cf9de7bae40e89de303acd6d